### DISCUSSION TITLE
Ella Diffusion

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/Fooocus#3138] [General]
https://github.com/TencentQQGYLab/ELLA
Another ai model project. This is Ella, Emma is coming too. Looks like we are gonna have more alternatives in future.

**Comment by xhoxye** [comment]
It doesn't have SDXL

**Reply by DavidDragonsage** [reply]
Ella SDXL has not been released yet, but they did have four demo images from it at the link.
